### ISSUE TITLE
Crash in ToastUtils showToast

### ISSUE THREAD

**Issue by loremattei** [wordpress-mobile/WordPress-Android#8590]
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getString(int)' on a null object reference
       at org.wordpress.android.util.ToastUtils.showToast(ToastUtils.java:25)
       at org.wordpress.android.ui.main.MeFragment$8.onLoadFailed(MeFragment.java:331)
       at org.wordpress.android.util.image.ImageManager$attachRequestListener$1.onLoadFailed(ImageManager.kt:277)
       at com.bumptech.glide.request.SingleRequest.onLoadFailed(SingleRequest.java:613)
       at com.bumptech.glide.request.SingleRequest.onLoadFailed(SingleRequest.java:593)
       at com.bumptech.glide.load.engine.EngineJob.handleExceptionOnMainThread(EngineJob.java:293)
       at com.bumptech.glide.load.engine.EngineJob$MainThreadCallback.handleMessage(EngineJob.java:327)
       at android.os.Handler.dispatchMessage(Handler.java:98)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5443)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
``` 

5ae5c2ea638393737aa7e57e-fabric

**Comment by rachelmcr** [comment]
Adding a high priority label as this is currently our most frequent crash.

**Comment by jtreanor** [comment]
Confirmed this is fixed in 11.6.
